packages/epo-widget-lib/src/widgets/SourceSelector/SourceSelector.tsx

```diff
@@ -0,0 +1,145 @@
+import {
+  FunctionComponent,
+  MouseEvent,
+  ReactNode,
+  useEffect,
+  useState,
+} from "react";
+import { Alert, Source } from "@/types/astro";
+import { Image } from "@rubin-epo/epo-react-lib/dist/types/image";
+import { useTranslation } from "react-i18next";
+import { IconComposer } from "@rubin-epo/epo-react-lib";
+import * as Styled from "./styles";
+import Points from "./Points";
+import Message from "./Message";
+import { getLinearScale } from "./utilities";
+
+interface BlinkConfig {
+  autoplay?: boolean;
+  loop?: boolean;
+  duration?: number;
+}
+
+interface Elapsed {
+  day: number;
+  hour: number;
+}
+
+interface SourceSelectorProps {
+  width?: number;
+  height?: number;
+  sources: Source[];
+  alerts?: Alert[];
+  selectedSource?: Source[];
+  images: Image[];
+  selectionCallback?: (data: Source[]) => void;
+  blinkConfig?: BlinkConfig;
+  color?: string;
+}
+
+const SourceSelector: FunctionComponent<SourceSelectorProps> = ({
+  width = 600,
+  height = 600,
```

Comment:
domain has been removed along with accessors. Mapping astro units to pixels was a good idea but across investigations it was not actually implemented, `ra` and `dec` were simply pixel values. Instead we will use pixel values and the domain functions will be used to invert axes if necessary.